src/e2e.js

```diff
@@ -196,6 +196,30 @@ mye2e.prototype.dearmor = function(data) {
   return Promise.resolve(array2buf(e2e.openpgp.asciiArmor.parse(data).data));
 };
 
+mye2e.prototype.ecdhBob = function(curveName, peerPubKey) {
```

Comment:
Maybe an explanatory comment above this (something like "derive a shared secret using one-pass ECDH protocol").